### DISCUSSION TITLE
Frigate 0.14 Remove live mode from config

### DISCUSSION THREAD

**Discussion by lineumaciel** [blakeblackshear/frigate#11659] [Ask A Question]
Hi.

How to change the default mode from mse to webrtc. The live_mode option has been removed and we now have MSE by default.  ios 17.5 has an obvious problem with MSE despite the new API enabled in safari.

In theory, go2rtc should change from MSE to webrtc as it is in beta1. In the latest version, DEV stays with MSE.

**Comment by NickM-27** [comment]
Can you describe the problem more? It works as expected on my iPhone as well as another devs.

**Reply by lineumaciel** [reply]
Hi.

Today i spent some time figuring out the cause of the problem.  I am using a Tapo camera with two way audio in my test set up. When it is configured via go2rtc with two way audio.

  streams:
    source:
      - tapo://adminpass@192.168.1.2


Live stream MSE does not display video. When the microphone is turned on, the stream goes from MSE to Webrtc and everything works as expected.


Changing the configuration from tapo source to rtsp source solved the problem.

It seems to me that two-way audio causes problems with MSE. I will miss a bit the ability to set the default live mode.

**Comment by lineumaciel** [comment]
Hi

Yesterday I switched from version 0.14.0 Beta 1 to dev-8c32580, corrected the config and removed live_mode from the config.


On dev-8c32580 version I have a strange problem. 


In the UI(dashboard) itself, the image seems to me to refresh as expected. The problem occurs when i enter live from the camera. 
Safari strenuously tries to stay with the MSE stream, resulting in a black image. This is confirmed by go2rtc. When I click microphone the stream changes from MSE to webrtc and everything works as expected.


I checked it on several ios devices and the effect is the same everywhere.



Something escapes my attention but I don't know what. 


        {
            "type": "MSE/WebSocket active consumer",
            "remote_addr": "xxxx, xxxx, ::xxxxx",
            "user_agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.5 Mobile/15E148 Safari/604.1",
            "medias": [
                "video, sendonly, H264, H265",
                "audio, sendonly, MPEG4-GENERIC"
            ],
            "senders": [
                "255 H264, bytes=1005462, receivers=1",
                "97 MPEG4-GENERIC/8000, bytes=5225, receivers=1"
            ],
            "send": 978473
        }


![IMG_6418](https://github.com/blakeblackshear/frigate/assets/70479212/b860838f-38a2-4228-a57c-937fc08f6a69)
![IMG_6419](https://github.com/blakeblackshear/frigate/assets/70479212/fd67abc2-c6c9-4ae8-b8af-e4098871fcca)
